CMakeLists.txt

```
@@ -70,7 +70,9 @@ include(coverage)
 
 if(OLP_SDK_BUILD_EXTERNAL_DEPS)
     # Add third-party dependencies
+    set(EXTERNAL_BINARY_INSTALL_DIR ${CMAKE_CURRENT_BINARY_DIR}/install)
     add_subdirectory(external)
+    set(CMAKE_PREFIX_PATH ${EXTERNAL_BINARY_INSTALL_DIR})
```

I think you don't need to overwrite the existing `CMAKE_PREFIX_PATH`, which can be set externally by user. Better to add it via `list(APPEND)`:
```suggestion
    list(APPEND CMAKE_PREFIX_PATH ${EXTERNAL_BINARY_INSTALL_DIR})
```